SmartSaver.MVC/Controllers/DashboardController.cs

```diff
@@ -28,7 +32,23 @@ public IActionResult Index()
                 return View(nameof(Complete));
             }
 
-            return View(account);
+            Account acc = GetAccountAuth();
+
+            DashboardViewModel dvm = new DashboardViewModel()
+            {
+                SavedCurrentMonth = TransactionsCounter
+                    .AmountSavedCurrentMonth(_context.Transactions
+                    .Where(t => t.AccountId == acc.Id)),
```

Comment:
I believe passing account obejct's transactions would do the same thing: .AmountSavedCurrentMonth(Account.Transactions)